bin/lesshint

```diff
@@ -22,4 +23,6 @@ program
     .option('-x, --max-warnings [int]', 'Number of warnings to trigger nonzero exit code, defaults to 0')
     .parse(process.argv);
 
-cli(program);
+cli(program).always(function (status) {
+    exit(status.valueOf());
```

Comment:
I say we use `process.exitCode` instead. We only used `exit` to work around some strange output issue in an old Node version.